### ISSUE TITLE
Conflicts when not using json dataType

### ISSUE THREAD

**Issue by maximebedard** [victorquinn/Backbone.CrossDomain#16]
When using `Backbone.sync` with `dataType: 'jsonp'`, the library changes the dataType to json an tries to make a CORS call and get the following error on IE9.

![image](https://cloud.githubusercontent.com/assets/801405/7553016/e3964152-f6a6-11e4-8eef-6231eccb8e83.png)

Would it be usefull to early return when such a happen?

```js
if (options.dataType === 'jsonp') {
  return Backbone.vanillaSync.apply(this, arguments);
}
```

**Comment by mcallew** [comment]
+1

**Comment by mcallew** [comment]
i created a pull request fixing this 

https://github.com/victorquinn/Backbone.CrossDomain/pull/17

**Comment by victorquinn** [comment]
Fixed with #17 thanks @mcallew
